src/main/java/org/usfirst/frc4904/robot/RobotMap.java

```diff
@@ -2,92 +2,101 @@
 
 import org.usfirst.frc4904.standard.custom.controllers.CustomJoystick;
 import org.usfirst.frc4904.standard.custom.controllers.CustomXbox;
+import org.usfirst.frc4904.standard.custom.motioncontrollers.CANTalonSRX;
+import org.usfirst.frc4904.standard.subsystems.motor.Motor;
 
 import edu.wpi.first.wpilibj.kinematics.DifferentialDriveKinematics;
 
 public class RobotMap {
-    public static class Port {
-        public static class HumanInput {
-            public static final int joystick = 0;
-            public static final int xboxController = 1;
-        }
-
-        public static class CANMotor {
-        }
-
-        public static class PWM {
-        }
-
-        public static class CAN {
-        }
+  public static class Port {
+    public static class HumanInput {
+      public static final int joystick = 0;
+      public static final int xboxController = 1;
+    }
 
-        public static class Pneumatics {
-        }
+    public static class CANMotor {
+      // Climb
+      public static final int CLIMB_TUBE_MOTOR= -1;
+      public static final int CLIMB_WINCH_MOTOR = -1;
+    }
 
-        public static class Digital {
-        }
+    public static class PWM {
     }
 
-    public static class Metrics {
-        public static class Chassis {
-            public static final double TICKS_PER_REVOLUTION = -1; // TODO: CHANGE CONSTS
-            public static final double DIAMETER_INCHES = -1;
-            public static final double CIRCUMFERENCE_INCHES = Metrics.Chassis.DIAMETER_INCHES * Math.PI;
-            public static final double TICKS_PER_INCH = Metrics.Chassis.TICKS_PER_REVOLUTION
-                    / Metrics.Chassis.CIRCUMFERENCE_INCHES;
-            public static final double DISTANCE_FRONT_BACK = -1;
-            public static final double DISTANCE_SIDE_SIDE = -1;
-            public static final double INCHES_PER_TICK = Metrics.Chassis.CIRCUMFERENCE_INCHES
-                    / Metrics.Chassis.TICKS_PER_REVOLUTION;
-        }
+    public static class CAN {
     }
 
-    public static class DriveConstants {
-        public static final boolean kGyroReversed = false;
-        public static final double ksVolts = -1;
-        public static final double kvVoltSecondsPerMeter = -1;
-        public static final double kaVoltSecondsSquaredPerMeter = -1;
-        public static final double kTrackwidthMeters = -1;
-        public static final DifferentialDriveKinematics kDriveKinematics = new DifferentialDriveKinematics(
-                kTrackwidthMeters);
-        public static final double kPDriveVel = -1;
+    public static class Pneumatics {
     }
 
-    public static class AutoConstants {
-        public static final double kMaxSpeedMetersPerSecond = -1;
-        public static final double kMaxAccelerationMetersPerSecondSquared = -1;
-        public static final double kRamseteB = -1;
-        public static final double kRamseteZeta = -1;
+    public static class Digital {
+    }
+  }
+
+  public static class Metrics {
```

Comment:
non winch constants need not be in the winch branch (unless theyre for some reason needed for it to build)